config/initializers/twillio.rb

```diff
@@ -0,0 +1,6 @@
+require 'twilio-ruby'
+
+account_sid = ENV['TWILLIO_ACCOUNT_SID']
+auth_token = ENV['TWILLIO_AUTH_TOKEN']
```

Comment:
So we'll prob all need these set? Are these for the app (sort of like google oauth), or personal? Aka should we all sign up with Twilio for our own, or do you mind sharing?

Reply:
Right now they're not needed to pass any tests. Unsure of how to test twillio at the moment. You can find them in our Heroku console.